PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj

```diff
@@ -466,6 +476,7 @@
 		D0950AF71C618D86002E198C /* ViewControllers */ = {
 			isa = PBXGroup;
 			children = (
+				6F2A704A207EB56D00BF5AA0 /* filter */,
```

Comment:
Again, being nitpicky, but all the other groups are capitalized.

Reply:
Addressed in 8b4feb5